cbc-repeated-iv/src/cbc_game.rs

```diff
@@ -0,0 +1,528 @@
+    ▀▀▀▀▀▀▀▀▀▀   ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀    ▀         ▀         ▀  ▀▀▀▀▀▀▀▀▀▀▀"
+                    .to_string()
+            ),
+            4 => (
+                "Example with package that has the different name field being intercepted"
+                    .to_string(),
+                "Block #0 differs. We can assume that data in that block has changed. We don't \
+                know whether if data changed in other blocks or not. But if first block differs - \
+                we can guess that data structure is different this time...".to_string(),
+                "
+    ▄▄▄▄▄▄▄▄▄▄   ▄            ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄    ▄         ▄         ▄     ▄▄▄▄▄▄▄▄▄
+   ▐░░░░░░░░░░▌ ▐░▌          ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░▌  ▐░▌       ▐░▌       ▐░▌   ▐░░░░░░░░░▌
+   ▐░█▀▀▀▀▀▀▀█░▌▐░▌          ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀▀▀ ▐░▌ ▐░▌       ▄█░█▄▄▄▄▄▄▄█░█▄ ▐░█░█▀▀▀▀▀█░▌
+   ▐░▌       ▐░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌▐░▌       ▐░░░░░░░░░░░░░░░▌▐░▌▐░▌    ▐░▌
+   ▐░█▄▄▄▄▄▄▄█░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌░▌         ▀█░█▀▀▀▀▀▀▀█░█▀ ▐░▌ ▐░▌   ▐░▌
+   ▐░░░░░░░░░░▌ ▐░▌          ▐░▌       ▐░▌▐░▌          ▐░░▌           ▐░▌       ▐░▌  ▐░▌  ▐░▌  ▐░▌
+   ▐░█▀▀▀▀▀▀▀█░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌░▌         ▄█░█▄▄▄▄▄▄▄█░█▄ ▐░▌   ▐░▌ ▐░▌
+   ▐░▌       ▐░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌▐░▌       ▐░░░░░░░░░░░░░░░▌▐░▌    ▐░▌▐░▌
+   ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄▄▄ ▐░▌ ▐░▌       ▀█░█▀▀▀▀▀▀▀█░█▀ ▐░█▄▄▄▄▄█░█░▌
+   ▐░░░░░░░░░░▌ ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░▌  ▐░▌       ▐░▌       ▐░▌   ▐░░░░░░░░░▌
+    ▀▀▀▀▀▀▀▀▀▀   ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀    ▀         ▀         ▀     ▀▀▀▀▀▀▀▀▀"
+                    .to_string()
+            ),
+            _ => (
+                "What are you doing here, step-bro??? UwU".to_string(),
+                "Okay, here's some small secret crypto-tip for you:".to_string(),
+                "https://www.youtube.com/watch?v=dQw4w9WgXcQ".to_string()
+            )
+        }
+    }
+
+    // Creating beautified ascii table
+    // User must send strings (lines), length of each line and padding
+    // We need to send length of each line explicitly, because ColoredString adding extra characters
+    // That are hidden in standard console output
+    fn create_table(
+        strings_arr: &[&String],
+        strings_length: &[usize],
+        padding: usize,
+    ) -> anyhow::Result<String> {
+        let mut result_string = String::new();
+        let longest_string = strings_length
+            .iter()
+            .max()
+            .expect("Can not retrieve max value from strings_length array");
+        let size = longest_string + padding * 2;
+
+        writeln!(result_string, "╔{}╗", "═".repeat(size)).expect("");
+        writeln!(result_string, "║{}║", " ".repeat(size)).expect("");
+        for string_index in 0..strings_arr.len() {
```

Comment:
You could replace index-based loop with an iterator-based one
```suggestion
        for (string, length) in string_arr.iter().zip(strings_length) {
```